docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+- **Versioning** - Atom's last (at the time) release was Atom 1.60, which left us with some issues. We wanted to stay with [Semantic Versioning](https://semver.org/) as Atom had, but what should our first full release be? We could reset the counter back to 1.0, but that would cause packages looking for a minimum version of Atom to break. We could move to 2.0, but again, there was an issue with packages, which were specifically looking only for 1.x versions, but this time as a *maximum* version. We could continue the numbering from Atom, but this seemed like we weren't getting a clean break as a new project. In the end, we decided to jump from 1.60 to 1.100 - a number that would give us a clean start whilst avoiding the issues with package version limits.
+- **Electron updates** - This was, and still is, one of our biggest goals for Pulsar. When development slowed in Atom, it caused the Electron version to get further and further behind. At the time Pulsar was born, it was using Electron 9, which went EOL in early 2021. We managed to get Pulsar onto Electron 12, but to go further required some rather extensive changes that are still being worked on. Read our blog post on [The Quest for Electron LTS](https://pulsar-edit.dev/blog/20240124-mauricioszabo-the-quest-for-electron-lts.html) for more information on the topic.
+
+With this work either complete or underway, we made our first tagged release of Pulsar - [1.100.0](https://github.com/pulsar-edit/pulsar/releases/tag/v1.100.0-beta) - on the same day that Atom was finally sunset - the 15th of December 2022.
+
+### Website, documentation, and socials
+
+<img src="./assets/website.png" height="250" />
+
+We needed to show the world that Pulsar was a serious project in order to make sure that people looking for Atom alternatives (and those looking for VS Code or other text editor alternatives) knew we were around and were a legitimate option. To this end, we created the [pulsar-edit.dev](https://pulsar-edit.dev) website to serve as a hub for our download links, news and blog posts, documentation, and links to social platforms for people to get help.
+
+We archived a copy of the original Atom flight manual as well as creating an edited, rebranded, and updated version of it to work with our website framework to make sure that we had documentation we could link to and maintain. We made a download page for people to find the latest releases easily, rather than relying purely on a GitHub release link. This was a substantial amount of work that was split between multiple people and took a while to complete. Could we have *technically* got by without this? Probably, but we wanted to show the whole community that we were deadly serious about our commitment to the project and wanted to make sure that it came across this way to anyone who stumbled upon us.
```

Comment:
```suggestion
We archived a copy of the original Atom Flight Manual as well as creating an edited, rebranded, and updated version of it to work with our website framework to make sure that we had documentation we could link to and maintain. We made a download page for people to find the latest releases easily, rather than relying purely on a GitHub release link. This was a substantial amount of work that was split between multiple people and took a while to complete.

Could we have *technically* got by without this? Probably, but we wanted to show the whole community that we were deadly serious about our commitment to the project and wanted to make sure that it came across this way to anyone who stumbled upon us.
```